graphbolt/src/csc_sampling_graph.cc

```diff
@@ -80,6 +80,21 @@ void CSCSamplingGraph::Load(torch::serialize::InputArchive& archive) {
     type_per_edge_ =
         read_from_archive(archive, "CSCSamplingGraph/type_per_edge").toTensor();
   }
+  if (read_from_archive(archive, "CSCSamplingGraph/has_edge_attributes")
```

Comment:
We need an option to decide whether to load edge attributes. Also add it in python side.

Reply:
Right~ I'll add the load option flag in next PR.